lib/typing/parquet.go

```diff
@@ -69,18 +68,12 @@ type Field struct {
 }
 
 func (k *KindDetails) ParquetAnnotation(colName string) (*Field, error) {
-	var stringKind bool
-
-	// If it's a date or time, it should be a STRING annotation.
-	if k.Kind == Date.Kind {
-		stringKind = true
-	}
-
-	if k.ExtendedTimeDetails != nil && k.ExtendedTimeDetails.Type == ext.TimeKindType {
-		stringKind = true
-	}
-
-	if k.Kind == String.Kind || k.Kind == Struct.Kind || stringKind {
+	switch k.Kind {
```

Comment:
Can this be combined with the `switch k.Kind` below?

Reply:
Yes it can :)